storagemarket/README.md

```diff
@@ -0,0 +1,203 @@
+# How To Use the StorageMarket module
```

Comment:
Same comment as on retrieval market. Flesh out with high level description and then a brief how-to construct and use the module.

Reply:
Make sure to cover all the dependencies on the storage and provider side including the ones that are provided by this module but must still be constructed by the node.

I think the GFC code is actually a pretty good reference for this if you want to construct and example.

Reply:
@hannahhoward I think I got them all.